recipes/minigun/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "minigun" %}
+{% set version = "0.0.1.dev0" %}
```

Comment:
Is there a fully released version to build from?

Reply:
Ideally, we should wait to have a git tag before merging this. See https://github.com/dglai/minigun/issues/14